docs/guides/enqueue.md

```diff
@@ -3,4 +3,111 @@ id: enqueue
 title: Enqueue
 ---
 
-Content will be added soon...
+Manifest class is located in `Eightshift Libs`. To extend it, use one of the following classes:
+* `Eightshift_Libs\Enqueue\Enqueue_Admin` class.
+* `Eightshift_Libs\Enqueue\Enqueue_Blocks` class.
+* `Eightshift_Libs\Enqueue\Enqueue_Theme` class.
+
+These classes are made to create Javascript and Css files of the project for every project part like Admin area, Block Editor area of Theme area.
+
+The usage of any of these classes is optional, and you can use only what you need. Eightshift Boilerplate comes with all three classes all-ready implemented.
+
+Each of these classes implements have some actions that can be deregistered using the standard [WordPress way](https://developer.wordpress.org/reference/functions/remove_action/).
+
+Enqueue classes work in combination with [webpack build](/eightshift-docs/docs/advanced/webpack) part for your project.
+
+File names are defined inside a class variable that can be changed by extending the class and providing the override for the variable. If you change the file names, you must provide changes to the webpack build process.
```

Comment:
```suggestion
File names are defined inside a class variable that can be changed by extending the class and providing the override for the variable. If you change the file names, you must provide changes to the webpack build process.
```

Should the variables actually be constants? I'm looking at the parts below and it seems to me like it should be constants instead of variables.